src/meta/Matches.ts

```diff
@@ -0,0 +1,14 @@
+import { Base } from './Base';
+
+export default class Matches extends Base {
+	/**
+	 * Determine if the target of a particular `Event` matches the virtual DOM key
+	 * @param key The virtual DOM key
+	 * @param event The event object
+	 */
+	public get(key: string, event: Event): boolean {
+		this.requireNode(key);
```

Comment:
Okay what if the `node1` one does exist and the successful user action causes the node to not be included (correctly due to the user action) in the next `render`, it would throw an error.... The seems a bit confusing to me (but it is the same for Dimensions too)